packages/nimble-components/src/rich-text-editor/tests/rich-text-editor.stories.ts

```diff
@@ -50,6 +58,16 @@ Note: The content in the \`footer-actions\` slot will not adjust based on the st
 client application to make any necessary adjustments. For example, if the buttons should be disabled when the rich-text-editor is disabled, the
 client application must implement that functionality.
 `;
+const fitToContentDescription = `Setting \`fit-to-content\` allows the editor to grow vertically to fit the content instead of enabling the
+vertical scrollbar when content reaches the default height.
+
+To observe the changes when toggling, add more than five lines in the editor; this will enable the vertical scrollbar to view the hidden content.
+If the \`fit-to-content\` option is enabled, the editor will grow vertically to accommodate the content, instead of displaying the vertical scrollbar.`;
```

Comment:
```suggestion
const fitToContentDescription = `Setting \`fit-to-content\` allows the editor to grow vertically to fit the content instead of displaying a
vertical scrollbar when content exceeds the visible height.`;
```